beacon/goclient/goclient.go

```diff
@@ -475,3 +477,70 @@ func (gc *GoClient) slotStartTime(slot phase0.Slot) time.Time {
 	startTime := time.Unix(gc.network.MinGenesisTime(), 0).Add(duration)
 	return startTime
 }
+
+func (gc *GoClient) Events(ctx context.Context, topics []string, handler eth2client.EventHandlerFunc) error {
+	if err := gc.multiClient.Events(ctx, topics, handler); err != nil {
+		gc.log.Error(clResponseErrMsg,
+			zap.String("api", "Events"),
+			zap.Error(err),
+		)
+
+		return err
```

Comment:
as far as I know this should be already in `stage` branch, done by @oleg-ssvlabs in #2018

Reply:
It seems I kept it by mistake while merging. Deleted